helm/postgres/values.yaml

```diff
@@ -158,10 +158,13 @@ postgresVersion: 14
 # standby sets whether or not to run this as a standby cluster. Both of the
 # values below are required to enable a standby cluster. Setting "enabled" to
 # "true" eunables the standby cluster while "repoName" points to a pgBackRest
-# archive to replay WAL files from.
+# archive to replay WAL files from, and "host" and "port" point to a primary
+# cluster from which to stream data.
 # standby:
 #   enabled: false
 #   repoName: repo1
+#   host: "<primary-ip>"
```

Comment:
As with the other examples, could you adjust the `repoName` and `host` value values to better match what the actual values would be? For example,
```
#   host: "1.2.3.4"
#   port: 5432
```

Reply:
Sure, adjusting `host` to show an RFC-5737 address and `port` to specify `5432`.